docs/_sources/comet.rst.txt

```diff
@@ -0,0 +1,49 @@
+.. _comet:
+
+Comet Integration
+===========================
+
+
+CodeCarbon automatically integrates with `Comet <https://www.comet.ml/site/>`_ for experiment tracking and visualization. Comet provides data scientists with powerful tools to track, compare, explain and reproduce their experiments, and now with CodeCarbon you can easily track the carbon footprint of your jobs along with your training metrics, hyperparameters, dataset samples, artifacts and more.
+
+.. image:: ./images/comet-workspace.png
```

Comment:
@sashavor can you move the `.png` files in `docs/_images` to `docs/edit/images` for that is the path you are referring to here, which will get populated eventually in `docs/_images` as Sphinx will make those renderings on its own.